From 978a8d78493a497e52b22a507b75b2de4b809f58 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Fri, 29 Sep 2023 14:31:10 +0200
Subject: [PATCH 0564/1141] arm64: defconfig: enable USB_HIDDEV

Enable devices with a separate event interface on /dev/usb/hiddevX

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I2fb619747c66329d25d7375e34d3540d9daf6686
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/330608
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -99,6 +99,7 @@ CONFIG_TYPEC_UCSI=m
 CONFIG_UBIFS_FS=y
 CONFIG_UCSI_STM32G0=m
 CONFIG_USB_DWC3_STM32=y
+CONFIG_USB_HIDDEV=y
 CONFIG_USB_STM32_USBH=y
 CONFIG_V4L_TEST_DRIVERS=y
 CONFIG_VIDEOMODE_HELPERS=y
